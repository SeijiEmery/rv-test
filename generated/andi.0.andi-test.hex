0000000    13  f2  11  00  93  f2  f1  7f  13  f3  f1  0f  93  f3  01  00
0000020    73  00  10  00                                                
0000024
