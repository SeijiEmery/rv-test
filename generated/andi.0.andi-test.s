andi    x4, x3, 1
    andi    x5, x3, 0x7FF
    andi    x6, x3, 0xFF
    andi    x7, x3, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
ebreak
