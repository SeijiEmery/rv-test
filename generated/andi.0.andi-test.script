load /x 0 generated/andi.0.andi-test.hex
writereg 3 0
writereg 4 0
writereg 5 0
writereg 6 0
writereg 7 0
setpc 0
run 1004
readreg 3
readreg 4
readreg 5
readreg 6
readreg 7