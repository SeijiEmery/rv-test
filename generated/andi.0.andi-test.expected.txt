tests/andi.test.s:2
R3 = 0
R4 = 0
R5 = 0
R6 = 0
R7 = 0
